### PR TITLE
EPRS: Removing EOL operating systems from java support page

### PR DESCRIPTION
**Pull request by gvasquezvargas**
## What Changed?

https://enterprisedb.atlassian.net/browse/DOCS-921

### CODE REVIEW

File: product_docs/docs/eprs/7/supported_platforms.mdx [hunk 1]
```diff
@@ -9,17 +9,16 @@ Replication Server is certified to work with the following Java platforms:
...
-| RHEL 7 and 8               | -   Red Hat OpenJDK 8     <br /> <br /> -   Oracle JDK 8     <br /> |
+| RHEL 8                     | -   Red Hat OpenJDK 8     <br /> <br /> -   Oracle JDK 8     <br /> |
 | RHEL 9                     | Red Hat OpenJDK 11                                                  |
 | Rocky Linux/AlmaLinux 8    | Red Hat OpenJDK 8                                                   |
 | Rocky Linux 9 /AlmaLinux 9 | Red Hat OpenJDK 11                                                  |
 | PPCLE RHEL 8               | Red Hat OpenJDK 8                                                   |
 | PPCLE RHEL 9               | Red Hat OpenJDK 11                                                  |
 | Windows Server 2022        | Oracle JDK 8                                                        |
-| Debian 10 and 11           | Red Hat OpenJDK 11                                                  |
+| Debian 11                  | Red Hat OpenJDK 11                                                  |
```
Comment (gvasquezvargas) [inline product_docs/docs/eprs/7/supported_platforms.mdx:21]: Just double-checking: 
Is support for Debian 12 going to be added for the upcoming release?
If yes, we'll need to update the installation templates and update this table with the JDK value for Debian 12.

Comment (gvasquezvargas) [reply product_docs/docs/eprs/7/supported_platforms.mdx:21]: Correction: We added the installation docs for Debian 12 yesterday in https://github.com/EnterpriseDB/docs/pull/5924, so I guess the question is: 
Which JDK version do we want to provide here for Debian 12? 
Red Hat OpenJDK 11 ?

Comment (gvasquezvargas) [reply product_docs/docs/eprs/7/supported_platforms.mdx:21]: [XDB 16.08.24 daily sync]

Debian 12 is shipped with Red Hat OpenJDK 17. 

But we have not tested this yet, so we won’t make any updates for this now.

**Review by zahid75 — APPROVED** [review]
Changes look fine.

**Review by djw-m — APPROVED** [review]
LGTM

**Pull request merged**